### DISCUSSION TITLE
aws : Permission denied (publickey)

### DISCUSSION THREAD

**Discussion by cyphos1973** [deployphp/deployer#3404] [Help needed]
Hello
I tried with my ssh key but it' dont work
There is the error

`[aws] err ubuntu@ec2-xx-xx-xx-xx.eu-west-3.compute.amazonaws.com: Permission denied (publickey).
[aws] exit code -1 (Unknown error)`

` host('aws')
        ->setHostname('ec2-xx-xx-xx-xx.eu-west-3.compute.amazonaws.com')
        ->setPort('22')
        ->setRemoteUser('ubuntu')
        ->set('identityFile','~/.ssh/aws.pem')
        ->set('deploy_path', '/var/www/html/test-deploy')
        ;`

Thanks you a lot for your responses

**Comment by antonmedv** [comment]
Use _ insert of camel case.

**Comment by cyphos1973** [comment]
`->set('deployPath', '/var/www/html/test-deploy') ; `
exact??

**Reply by antonmedv** [reply]
With _
